### DISCUSSION TITLE
Lists under Repositories

### DISCUSSION THREAD

**Discussion by FallenDev** [community/community#31810] [Other Feature Feedback, Questions, & Ideas]
The new beta feature of Lists is pretty awesome, I like being able to categorize past projects I've starred. I have a suggestion to also either create the capability to make lists under repositories or also have a link there from the lists created in stars. Since I personally feel I visit the stars tab less than I do the repositories tab.

**Comment by jfullstackdev** [comment]
if it's on your end, you can do that manually by putting the link of the list on the README of the repo, so another user can check further projects that are related to that
